samples/multicluster/gen-eastwest-gateway.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+#
+# Copyright Istio Authors
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+set -euo pipefail
+
+if [[ -n "${CLUSTER:-}" ]]; then
+  echo The CLUSTER environment variable must be set.
+  exit 1
+fi
+
+if [[ -n "${NETWORK:-}" ]]; then
+  echo The NETWORK environment variable must be set.
+  exit 1
+fi
+
+# Generate the YAML for the east-west gateway.
+istioctl manifest generate -f - <<EOF
```

Comment:
Should we allow generating the below yaml/show it to the users to help those who want to manage their gateway deployment with operator? They could `kube create -f` the spec below to add the gateway without messing with their istio installation. 

If we expose the below yaml to users, we should probably note that using `install` instead of `manifest generate | kubectl ...` will reconcile and remove your already deployed components. Doing it the way you have here should help prevent users from doing that..

Reply:
@stevenctl 

The yaml is exposed to users ... after running this script, the yaml will be dumped to stdout. The caller can do whatever they want with it (e.g. redirect to a file of their choosing).

We should have some instructions about using manifest install with gateways, but I'm just not sure where it should go. Maybe here?